pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -675,16 +675,12 @@ func (k *Bootstrapper) JoinCluster(cc config.ClusterConfig, n config.Node, joinC
 
 		out, err := k.c.RunCmd(exec.Command("/bin/bash", "-c", joinCmd))
 		if err != nil {
-			if strings.Contains(err.Error(), "status \"Ready\" already exists in the cluster") {
-				klog.Infof("Node %s already joined the cluster, skip failure.", n.Name)
-			} else {
-				return errors.Wrapf(err, "cmd failed: %s\n%+v\n", joinCmd, out.Output())
-			}
+			return errors.Wrapf(err, "cmd failed: %s\n%+v\n", joinCmd, out.Output())
```

Comment:
RunCmd will include the rr.output in it, so no need to add that
and also no need for \n

suggestion

check for the error 
```
already exists in the cluster. You must delete the existing Node or change the name of this new joining Node
```
if that error is there, we will klog.Info (so the IDE users who usually dont like to wait without any logs, know we are retrying
```
if rr.Output() has that ^^ error, {
klog.Info("retrying to re-register worker node to the controlplane.... ")
}

```

Reply:
also, RunCmd already includes the rr.Output in the error message,
return errors.Wrapf(err, "join cmd") is enough